libfwupdplugin/fu-hwids-fdt.c

```diff
@@ -92,6 +93,20 @@ fu_hwids_fdt_setup(FuContext *ctx, FuHwids *self, GError **error)
 		fu_fdt_image_get_attr_str(FU_FDT_IMAGE(fdt_img), "version", &version, NULL);
 		fu_hwids_add_value(self, FU_HWIDS_KEY_BIOS_VERSION, version);
 	}
+	fdt_img_backplane =
+	    fu_fdt_firmware_get_image_by_path(FU_FDT_FIRMWARE(fdt),
+					      "vpd/root-node-vpd@a000/enclosure@1e00/backplane@800",
+					      NULL);
+	if (fdt_img_backplane != NULL) {
```

Comment:
This is always returning null

Note that the `fu_fdt_firmware_get_image_by_path(FU_FDT_FIRMWARE(fdt), "ibm,firmware-versions", NULL);` line above is also returning NULL on the blackbird with both the copy I built and the fwupd version in apt.

Reply:
Hmm, can you do a `sudo ./src/fwupdtool hwids -vv` log please.

Reply:
The bianry is located at `/home/meklort/git/fwupd/venv/bin/fwupdtool` following the latest readme, but...
[hwids.txt](https://github.com/user-attachments/files/16155392/hwids.txt)

Reply:
Aha, a missing prefix slash. Can you try the branch again please.